base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
I think the correct name would be NT. I'll change our make file to use that instead and flow the changes through to here. WindowsNT, Windows_NT, WinNT, winnt, and WINNT all seem to be common variants though, so I had gone with our Makefile normalization initially